src/main/resources/db/migration/V3__tilkjent_ytelse.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE tilkjent_ytelse (
+    id                                  BIGSERIAL          primary key,
```

Comment:
Siden id brukes i api, så bør den ikke være sekvensiell. Bruk uuid, slik som på sak.

Reply:
Slet med å få UUID til å fungere med @Id av en eller annen mystisk grunn. Trenger et løpenummer for internbruk, så jeg behold sekvens som primærnøkkel, men lager en ekstern id som brukes fra controller.

Reply:
Det hadde vært fint om vi kunne hatt samme primærnøkkel på alle tabellene våre. Det fungerer jo med uuid som primærnøkkel på de andre tabellene i ef-sak. Hvis problemet var med save, så ligger det en generisk CustomRepository under familie.ef.sak.repository som implementerer persist.